lib/formatter/csv/annotation_for_csv.rb

```diff
@@ -133,66 +140,40 @@ def dropdown_find_selected_option(selects, answer_value)
       end
 
       def task_label(task_info)
-        translate(task_info["question"] || task_info["instruction"])
+        workflow_information.string(task_info['question'] || task_info['instruction'])
       end
 
       def tool_label(task_info, tool)
         tool_index = tool["tool"]
         have_tool_lookup_info = !!(task_info["tools"] && tool_index)
         known_tool = have_tool_lookup_info && task_info["tools"][tool_index]
-        translate(known_tool["label"]) if known_tool
+        workflow_information.string(known_tool['label']) if known_tool
       end
 
       def answer_labels
         Array.wrap(@current["value"]).map do |answer_idx|
           begin
             task_answer = workflow_at_version.tasks[@current['task']]['answers'][answer_idx]
             answer_string = task_answer['label']
-            translate(answer_string)
+            workflow_information.string(answer_string)
           rescue TypeError, NoMethodError
             "unknown answer label"
           end
         end
       end
 
-      def translate(string)
-        @translations ||= workflow_at_version.strings
-        @translations[string]
-      end
-
       def workflow_at_version
-        cache.workflow_at_version(classification.workflow, workflow_version, content_version)
-      end
-
-      def workflow_version
-        classification.workflow_version.split(".")[0].to_i
-      end
-
-      def content_version
-        classification.workflow_version.split(".")[1].to_i
-      end
-
-      def annotation_by_task(subtask)
-        workflow_at_version.tasks.find do |key, task|
-          key == subtask['task']
-        end
+        workflow_information.at_version
       end
 
-      def get_task_info(subtask)
-        workflow_at_version.tasks.find do |key, task|
-         key == subtask["task"]
-        end.try(:last) || {}
-      rescue
+      # used in combo task to protect against invalid annotations
+      # i.e. bad annotation data from tropical sweden project
```

Comment:
This comment doesn't add a lot of context--what's a tropical sweden? Is there a better way to phrase this?

Reply:
this has a relevant spec https://github.com/zooniverse/panoptes/blob/5928d94cffb424e68aec480c619271d0783bb0dc/spec/lib/formatter/csv/annotation_for_csv_spec.rb#L196-L210
```suggestion
      # i.e. bad annotation data from tropical sweden project
      # see https://github.com/zooniverse/panoptes/blob/5928d94cffb424e68aec480c619271d0783bb0dc/spec/lib/formatter/csv/annotation_for_csv_spec.rb#L196-L210
```